doc/source/train/getting-started-pytorch-lightning.rst

```diff
@@ -190,6 +190,13 @@ Begin by wrapping your code in a :ref:`training function <train-overview-trainin
 
 Each distributed training worker executes this function.
 
+You can specify the input argument for `train_func` via the Trainer's `train_loop_config` parameter.
```

Comment:
Optionally, we can extract this section out to a separate file and include it, similar to what's being done [here](https://github.com/ray-project/ray/blob/c67de894a3aedf2621a118eb95236a04ab9110f1/doc/source/train/getting-started-pytorch.rst?plain=1#L321).

In the future we may just have a full separate user guide for this.